IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/user_settings.h

```diff
@@ -60,8 +61,10 @@
 #define FREERTOS
 #define FREERTOS_TCP
 
-
-
+#if !defined(FREERTOS_TCP)
+ #define WOLFSSL_NO_SOCK
```

Comment:
Indent should be 4 spaces here and in line below.

Reply:
Thank you for pointing it out. Fixed

In addition to that, I update Copyright year